src/backend/distributed/sql/udfs/get_cluster_clock/latest.sql

```diff
@@ -0,0 +1,20 @@
+ALTER TABLE pg_catalog.pg_dist_local_group ADD COLUMN logical_clock_value BIGINT NOT NULL DEFAULT 0;
+
+UPDATE pg_catalog.pg_dist_local_group SET logical_clock_value = extract(epoch from now()) * 1000;
+
+DROP FUNCTION IF EXISTS pg_catalog.get_cluster_clock();
```

Comment:
I guess we don't need this ?

Reply:
@onurctirtir you meant the DROP is needed?

Reply:
Ideally we don't need it, it's a little gray area to me. If I check other scripts, they seem to have it?

Reply:
> @onurctirtir you meant the DROP is needed? 

yes

>  If I check other scripts, they seem to have it?

I guess the ones that update the arg/return types have, not the ones defining a brand new function

Reply:
Got it, removed now.